content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -628,6 +628,47 @@ So a webhook response to add that label would be:
 {{% /tab %}}
 {{< /tabs >}}
 
+Starting in v1.19, admission webhooks can optionally return warning messages that are returned to the requesting client.
+Warnings can be sent with allowed or rejected admission responses.
+No "Warning:" prefix should be included in the message.
+Warning messages should describe a problem the client making the API request should correct or be aware of.
+Limit warnings to 120 characters if possible. Warnings over 256 characters and large numbers of warnings may be truncated.
```

Comment:
It took me a long time to work out the intended meaning. I think this is equivalent.

```suggestion
If you're implementing a webhook that returns a warning:

- don't include the header name `Warning:` in the message
- write messages that describe the problem the client making the API request should correct or be aware of
- limit messages to 120 characters if possible.

{{< caution >}}
The API server might truncate a warning message that exceeds 256 characters.  
Also, if one or more webhooks issue several warnings that combine to exceed 256 characters, the API server might truncate a shorter message that takes the combined warning-text length over the limit.
{{< /caution >}}
```

Reply:
that's not quite equivalent, but I tried to adapt to match the suggested structure

Reply:
Can you correct:
`write messages that describe the problem the client making the API request should correct or be aware of`
This is hard to parse.